table/block_based/block_based_table_reader_test.cc

```diff
@@ -722,6 +722,199 @@ TEST_P(ChargeTableReaderTest, Basic) {
   }
 }
 
+class StrictCapacityLimitReaderTest : public BlockBasedTableReaderTest {
+ public:
+  StrictCapacityLimitReaderTest() : BlockBasedTableReaderTest() {}
+
+ protected:
+  void ConfigureTableFactory() override {
+    BlockBasedTableOptions table_options;
+
+    table_options.block_cache = std::make_shared<
+        TargetCacheChargeTrackingCache<CacheEntryRole::kBlockBasedTableReader>>(
+        (NewLRUCache(4 * 1024, 0 /* num_shard_bits */,
+                     true /* strict_capacity_limit */)));
+
+    table_options.cache_index_and_filter_blocks = false;
+    table_options.filter_policy.reset(NewBloomFilterPolicy(10, false));
+    table_options.partition_filters = true;
+    table_options.index_type = BlockBasedTableOptions::kTwoLevelIndexSearch;
+
+    options_.table_factory.reset(NewBlockBasedTableFactory(table_options));
+  }
+};
+
+TEST_P(StrictCapacityLimitReaderTest, StrictCapacityLimitGet) {
+  // Task T205994926: Test that we get error status when we exceed
```

Comment:
Nit: We don't need to mention the task here

Reply:
Updated PR